lib/index.js

```diff
@@ -365,6 +343,24 @@ function stringToBuffer(string) {
 }
 
 /**
+ *
+ * Converts an ArrayBuffer to a Buffer
+ *
+ * @api private
+ */
+
+function arrayBufferToBuffer(data) {
```

Comment:
I was wrong. Actually `byteOffset` seems necessary. 
Anyway I think we can fix that problem as other PR.